scss/bootstrap/mixins/_page-header-variant.scss

```diff
@@ -0,0 +1,8 @@
+@mixin page-header-variant($bg-color) {
+  background: $bg-color;
+
+  > h1,
+  > h2 {
```

Comment:
Yeah, i was on the fence about it, wanted to see if you had an opinion on this. I'll add the other heading levels